messages/3.0.0.md

```diff
@@ -0,0 +1,206 @@
+# MarkdownTOC - 3.0.0
+
+---
+
+**NOTE**
+This upgrade contains some **breaking changes**. 
+Please follow "Upgrade guide from v2 to v3" instructions.
```

Comment:
Suggestion: "Please follow "Upgrade guide from v2 to v3" for instructions."

Reply:
I see.